src/databricks/labs/ucx/runtime.py

```
@@ -23,6 +23,78 @@
 logger = logging.getLogger(__name__)
 
 
+def _get_view_definition(hms_db: str) -> str:
+    return f"""CREATE OR REPLACE VIEW hive_metastore.{hms_db}.vw_failure_summary
+    AS
+    WITH failuretab (failure, issue, object_type) AS (
+      SELECT
+          explode(from_json(failures, 'array<string>')) AS failure,
+          substring_index(failure, ":", 1) issue,
+          object_type
+        FROM
+        (
+        SELECT
+          failures,
+          "jobs" AS object_type
+        FROM
+          hive_metastore.{hms_db}.jobs
+        WHERE failures IS NOT NULL and failures != '[]'
+        UNION ALL
+        SELECT
+          failures,
+          "clusters" AS object_type
+        FROM
+          hive_metastore.{hms_db}.clusters
+        WHERE failures IS NOT NULL and failures != '[]'
+        UNION ALL
+        SELECT
+          failures,
+          "global init scripts" AS object_type
+        FROM
+          hive_metastore.{hms_db}.global_init_scripts
+        WHERE failures IS NOT NULL and failures != '[]'
+        UNION ALL
+        SELECT
+          failures,
+          "pipelines" AS object_type
+        FROM
+          hive_metastore.{hms_db}.pipelines
+        WHERE failures IS NOT NULL and failures != '[]'
+        UNION ALL
+        SELECT
+          concat('["', CASE
+           WHEN STARTSWITH(location, "wasb") THEN "Unsupported Storage Type"
+           WHEN STARTSWITH(location, "adl") THEN "Unsupported Storage Type"
+       END , '"]') AS failures,
+          IF(object_type IN ("MANAGED", "EXTERNAL"), "Table", "View") AS object_type
+        FROM
+          hive_metastore.{hms_db}.tables
+      WHERE location IS NOT NULL AND location != ""
+      UNION ALL
+      SELECT concat('["', CASE
+           WHEN STARTSWITH(error, "ignoring table because") THEN "Table Scan Failure"
```

This hides other errors